src/components/Notes/Notes.jsx

```diff
@@ -0,0 +1,113 @@
+import {
+  Box,
+  Flex,
+  Heading,
+  HStack,
+  SimpleGrid,
+  Text,
+  VStack,
+} from "@chakra-ui/react";
+import React, { useEffect, useState } from "react";
+
+import { updateCardById } from "../../actions/Card";
+import useUser from "../../lib/hooks/useUser";
+import AddNewNote from "./AddNewNote";
+import Note from "./Note";
+import { SentimentButton } from "./utils";
+
+const Notes = ({ cardId, notes, setCards }) => {
+  const [revNotes, setRevNotes] = useState(notes.map((n) => n).reverse());
+  const [newNote, setNewNote] = useState({ body: "", userId: "", date: "" });
+
+  const { user } = useUser();
+
+  useEffect(() => {
+    setRevNotes(notes.map((c) => c).reverse());
+  }, [notes]);
+
+  const handleSaveEdit = async (newNotes) => {
+    const updatedCard = await updateCardById(
+      cardId,
+      { notes: newNotes.map((n) => n).reverse() },
+      true
+    );
+    setCards((cards) => {
+      return cards.map((card) => {
+        if (cardId === card._id) {
+          return updatedCard;
+        } else {
+          return card;
+        }
+      });
+    });
+  };
+
+  const submitNewNote = async () => {
+    const newNotes = notes.concat(newNote);
+    const updatedCard = await updateCardById(
+      cardId,
+      {
+        notes: newNotes,
+      },
+      true
+    );
+
+    setCards((cards) => {
+      return cards.map((card) => {
+        if (cardId === card._id) {
+          return updatedCard;
+        } else {
+          return card;
+        }
+      });
+    });
+  };
+
+  return (
+    <SimpleGrid rows={2} gap={2} h="80vh" w="35%" p="5% 2% 5% 2%">
+      <VStack alignItems="left">
+        <Heading size="lg" mt={3} mb={2}>
+          Notes
+        </Heading>
+
+        <Box h="50vh" pr={1} overflowY="scroll">
+          <AddNewNote
+            newNote={newNote}
+            setNewNote={setNewNote}
+            submitNewNote={submitNewNote}
+          />
+
+          {revNotes.map((note, index) => {
```

Comment:
make a quick lil admin check using the ifAdmin from useUser hook. if the user is an admin, then render all possible notes. only if user is NOT admin, render the notes of that specific user

Reply:
i lied don't use ifAdmin function. Just literally check if user.isAdmin is true or not